CONTRIBUTING.md

```diff
@@ -214,7 +208,7 @@ CircleCI does not run them.
 
 ### Tests
 
-You can run 🤗 Transformers tests with `unittest` or `pytest`.
+You can run 🤗 Transformers (the underlying library for adapter) tests with `unittest` or `pytest`.
```

Comment:
This would not only run the HuggingFace tests but also some additional tests specific to adapter-transformers. Maybe this formulation is a bit misleading?

Reply:
Thanks for catching this @calpt ! 

I've added a short phrase to make it clear that both transformers and adapter-transformer tests will be run.